.codebuild/canary_workflow.yml

```diff
@@ -0,0 +1,59 @@
+ version: 0.2
+env:
+  shell: bash
+  compute-type: BUILD_GENERAL1_LARGE
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: build_linux
+      buildspec: .codebuild/build_linux.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+    - identifier: test
+      buildspec: .codebuild/test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+      depend-on:
+        - build_linux
+    - identifier: publish_to_local_registry
+      buildspec: .codebuild/publish_to_local_registry.yml
+      env:
+        compute-type: BUILD_GENERAL1_MEDIUM
+      depend-on:
+        - build_linux
+    - identifier: build_app_swift_us_east_1
+      buildspec: .codebuild/run_ios_modelgen_e2e_test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TEST_SUITE: src/__tests__/build-app-swift.test.ts
+          CLI_REGION: us-east-1
+      debug-session: true
+      depend-on:
+        - publish_to_local_registry
+    - identifier: build_app_android
```

Comment:
nit: keep the naming consistent b/w `build_app_swift_us_east_1` and `build_app_android`

Reply:
Yes will update the naming nit and regions accordingly